x-pack/solutions/observability/plugins/slo/server/services/get_slo_health.ts

```diff
@@ -55,12 +56,17 @@ export class GetSLOHealth {
         sloId: item.sloId,
         sloInstanceId: item.sloInstanceId,
         sloRevision: item.sloRevision,
+        sloName: item.sloName,
         state,
         health,
       };
     });
 
-    return fetchSLOHealthResponseSchema.encode(results);
+    /*
+     * Map results based on SLO ids since transforms represent all instances
+     */
+    const mappedResults = Array.from(new Map(results.map((item) => [item.sloId, item])).values());
```

Comment:
Sorry but I'm still not sure about this. 

We're returning only one element per `sloId`. What happens if we have different `revisions`? 
Should we return one element per revision in that case?

Also if the `instanceId` is not important I'd not return it.